### PR TITLE
Add default path prompt for deinit to delete etcd data

### PR DESCRIPTION
**Pull request by nuclearwu**
Signed-off-by: wuzhongjian <wuzhongjian_yewu@cmss.chinamobile.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Prompt the user to delete the default path of etcd data to deinit karmada

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/karmadactl/deinit.go [hunk 1]
```diff
@@ -297,6 +297,7 @@ func (o *CommandDeInitOption) Run() error {
 	}
 
 	fmt.Println("remove Karmada from Kubernetes successfully.\n" +
-		"\ndeinit will not delete etcd data, if the etcd data is persistent, please delete it yourself.")
+		"\ndeinit will not delete etcd data, if the etcd data is persistent, please delete it yourself." +
+		"\nthe default persistence path for etcd data is /var/lib/karmada-etcd")
```
Comment (lonelyCZ) [inline pkg/karmadactl/deinit.go:301]:
```suggestion
		"\nthe default persistence path for etcd data is '/var/lib/karmada-etcd'")
```

**Review by lonelyCZ — COMMENTED** [review]
Good prompt.

File: pkg/karmadactl/deinit.go (hunk 1 above)
Comment (nuclearwu) [reply pkg/karmadactl/deinit.go]: ok, it's been updated

**Review by RainbowMango — APPROVED** [review]
/approve
Looks good to me.

**Pull request merged**